pycheribuild/projects/syzkaller.py

```diff
@@ -62,6 +67,13 @@ def setup_config_options(cls, **kwargs):
             "run-sysgen", show_help=True,
             help="Rerun syz-extract and syz-sysgen to rebuild generated Go "
                  "syscall descriptions.")
+        cls.use_go = cls.add_config_option("use-go", kind=GOROOTOptions,
+            default=GOROOTOptions.DEFAULT, enum_choice_strings=[t.value for t in GOROOTOptions],
+            help="Specify GOROOT default is system")
```

Comment:
To make it consistent with `use-qemu`:
```
help="The Go type to run with."
```
The default value should be printed by `cheribuild --help` and doesn't have to be included in `help`.